pkg/odo/cli/component/component.go

```diff
@@ -59,25 +58,18 @@ func NewCmdComponent(name, fullName string) *cobra.Command {
 	var componentCmd = &cobra.Command{
 		Use:   name,
 		Short: "Components of an application",
-		Example: fmt.Sprintf("%s\n%s\n\n  See sub-commands individually for more examples, e.g. %s %s -h",
-			componentGetCmd.Example,
-			componentSetCmd.Example,
+		Example: fmt.Sprintf("%s\n%s\n\n  See sub-commands individually for more examples",
 			fullName, CreateRecommendedCommandName),
 		// 'odo component' is the same as 'odo component get'
 		// 'odo component <component_name>' is the same as 'odo component set <component_name>'
 		Run: func(cmd *cobra.Command, args []string) {
-			if len(args) > 0 && args[0] != GetRecommendedCommandName && args[0] != SetRecommendedCommandName {
-				componentSetCmd.Run(cmd, args)
```

Comment:
why was this removed @anmolbabu. due to this 
`odo component <get>` and `odo component python-asdf <set>` are not working

Reply:
yes that was intended

Reply:
as per workflow doc, odo get and odo set and accordingly odo component get and odo component set are deprecated

Reply:
so can we add deprecation info in the comments?

Reply:
We have discussed this. We will start gracefully deprecate after we have at least a beta version. We can remove it, let's just add a big warning to the release notes.

Reply:
ok